crates/next-custom-transforms/tests/errors/react-server-components/client-graph/root-params/output.stderr

```diff
@@ -0,0 +1,9 @@
+  x You're importing a component that needs "unstable_rootParams". That only works in a Server Component which is not supported in the pages/ directory. Read more: https://nextjs.org/docs/app/
```

Comment:
The PR is about disallowing in client components but the error is about attempting to use it in pages router. Is this just a limitation of how we’re testing it?

Reply:
yeah, the right message is inferred from context and all the tests basically assume pages router context rather than app router. I had the same initial reaction but it's how all the tests are currently set up and I don't want to do novel things here atm